app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,59 @@
+<div class="recipe-show">
+  <div class="top-links">
+    <%= link_to 'Back', recipes_path, class: "btn-back" %>
+    <% if @recipe.user_id == current_user.id %>
+      <%= link_to 'Add Food', new_food_url, :class => "btn-add"%>
+    <% end %>
+  </div>
+  <h2 class="recipe-name"><%= @recipe.name %></h2>
+  <div class="recipe-details">
+    <div class="">
+      <h3 class="">Preparation Time: <%= @recipe.preparation_time %>min(s)</h3>
+    </div>
+    <h3 class="">
+      Cooking Time: <%= @recipe.cooking_time %>min(s)
+    </h3>
+    <article class="">
+      <h3>Steps</h3>
+      <p><%= @recipe.description %></p>
+    </article>
+  </div>
+  <div class="recipe-toggler">
+    <% if @recipe.public %>
+      <p>Public</p>
+      <%= button_to "Change public to private", recipe_path(@recipe), method: :patch%>
+    <% else %>
+      <p>Private</p>
+      <%= button_to "Change public to public", recipe_path(@recipe), method: :patch%>
+    <% end %>
+  </div>
```

Comment:
- [ ] In order to match the given wireframe, let us kindly position the toggle button on the right side of the page as we can see below compared to ours
![Screenshot from 2023-04-21 15-22-17](https://user-images.githubusercontent.com/46388113/233676069-b8d96d7f-549b-4d99-9a1f-669385fc3152.png)
![Screenshot from 2023-04-21 15-22-39](https://user-images.githubusercontent.com/46388113/233676077-a3f60240-6930-4999-ab35-3a55b0897a93.png)

Reply:
Change noted and made.